challenges/2021_cloud_base_height/00_intro.ipynb

```diff
@@ -0,0 +1,49 @@
+{
```

Comment:
<p>I really like the idea of having this intro notebook, very nice.</p><p> </p><p>You might want to also make some mention of where the data can be found (so people can check they have access) and also what environments they are expected to have installed, so they can do that in preparation.</p><p> </p><p>Overall the series is looking really good, a nice logical progression with a sensible amount of useful description, this is going to be a really useful resource you have created.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MetOffice/data_science_cop/pull/30/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>